tests/suites/test_suite_bignum_mod.function

```diff
@@ -80,7 +80,196 @@ exit:
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+/* BEGIN_CASE */
+void mpi_residue_setup( )
+{
+    #define RS_ONE       0
+    #define RS_MAX_MIN1  1
+    #define RS_MAX       2
+    const char * s_data[ 3 ] = { "01", "fe", "ff" };
```

Comment:
Not sure why this is an array - might as well just have three `const char *` values here, with names `hex_01`, `hex_fe` and `hex_ff` (or something like that - I was thinking of `one` for the first, but that doesn't lend itself nicely to the other two)

(Note: `RS_MAX_MIN1` is not obviously "max minus 1", because "min" when juxtaposed with "max" commonly means "minimum")

Also note: replacing the array with variables will actually save one line of code!

Reply:
Yes, I was considering on how to make the tests more readable, and offloading it on the preproccessor was my choice. 

I do not feel strongly either way, but using variables will save more lines, because of the `#undefine`.